bigquery/google/cloud/bigquery/client.py

```diff
@@ -492,25 +493,45 @@ def load_table_from_storage(self, job_id, destination, *source_uris):
         """
         return LoadJob(job_id, destination, source_uris, client=self)
 
-    def copy_table(self, job_id, destination, *sources):
-        """Construct a job for copying one or more tables into another table.
+    def copy_table(self, sources, destination, **kwargs):
```

Comment:
You can use normal keyword arguments in this method.

The only reason I couldn't for extract table is that the destination there is a `*args` (varargs) and you have to do `**kwargs` for keyword arguments after a `*args` in Python 2.7.

Reply:
I wonder if I should change `extract_table` to just have the second argument be a single item or a list as you've done here? I kept with what was there, but this way of doing it makes more sense to me.

Reply:
Coming from Go, I like as little magic as possible, so I'd go with having the single-item-or-list.